### PR TITLE
Likes and comments

### PR DESCRIPTION
**Pull request by Rynvasis**
Add API POST `/posts/postId/like`
Add API PUT `/posts/postId/comment`

### CODE REVIEW

File: src/Controllers/Post.cs [hunk 1]
```diff
@@ -57,5 +57,31 @@ public IActionResult GetHomePosts()
...
+
+                return Ok();
+            }
+            catch (SecurityTokenException)
+            {
+                return Unauthorized();
+            }
+            catch (InvalidOperationException)
+            {
+                return BadRequest();
```
Comment (Salint) [inline src/Controllers/Post.cs:82]: Should throw a 409 Conflict instead.

**Review by Salint — CHANGES_REQUESTED** [review]
Also add ability to unlike a post.

**Comment by Salint** [comment]
Also return `likeCount`, `isLiked` and `comments` list when getting a post.

File: src/Controllers/Post.cs (hunk 1 above)
Comment (Rynvasis) [reply src/Controllers/Post.cs:82]: okay, I will solve it

**Review by Salint — APPROVED** [review]

**Pull request merged**